package.json

```diff
@@ -30,14 +30,14 @@
     }
   },
   "peerDependencies": {
-    "@lifeomic/chromicons": "^1.2.1",
+    "@lifeomic/chromicons": ">=1.2.1",
```

Comment:
Should this one be bumped higher as well?

Reply:
If we bump this to e.g >=2.2.0, it will unnecessarily cause warnings for consumers still using v1

Reply:
Ahh okay that makes sense